project.py

```diff
@@ -24,10 +25,26 @@ def newfunc(*args, **kwargs):
 
 
 class BoardSearcher:
-    def __init__(self):
+
+    last_saved_slide = None
+    last_saved_slide_mask = None
+
+    def __init__(self, n_slides=0,
+                 frame_counter=0,
+                 save_interval=30,
+                 similarity=0.70):
         self.width = None
         self.height = None
         self.saved_cnt = None
+        self.seed = None
+        self.number_of_slides = n_slides
+        self.frame_counter = frame_counter
+        # number of frames witch have to pass to run again check if
```

Comment:
@Foxiy I am not sure which witch is which but I do not think you meant that witch here.